scripts/mender.xml

```diff
@@ -3,9 +3,10 @@
   <default sync-j="4" revision="dunfell"/>
 
   <remote fetch="https://github.com/mendersoftware" name="mender"/>
+  <remote fetch="https://github.com/estape11" name="mender-temp"/>
```

Comment:
This needs to be removed before merging.

Reply:
Removed :D